Simplified/NYPLErrorLogger.swift

```diff
@@ -71,6 +71,10 @@ fileprivate let nullString = "null"
 
   // feeds
   case opdsFeedNoData = 800
+  
+  // DRM
+  case epubDecodingError = 900
```

Comment:
Could you change this to 1000? I used 900+ on develop and the hot fix branch.

Reply:
ok, done